src/Engine/Entity/World.cpp

```diff
@@ -85,10 +85,19 @@ Video::ParticleRenderer::Particle* World::GetParticles() const {
     return particles;
 }
 
+ENGINE_API unsigned int World::GetNrOfParticleSystems() const {
+
+    return nr_particle_sytems;
+}
+
 unsigned int World::GetParticleCount() const {
     return particleCount;
 }
 
+ENGINE_API void World::SetNrOfParticleSystems(unsigned int nrof_particleSystems) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'SetNrOfParticleSystems' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)